src/test/json/json-patch/add_something_test1.json

```diff
@@ -0,0 +1,13 @@
+[ {
```

Comment:
Based upon your path `SCIMPathPatter` are there any other examples we can add to this to test the new capability - or is this adequate?

Reply:
Added a test case for an enterprise property with a subattribute.